### PR TITLE
Update documentation to reflect feedback config

### PR DESCRIPTION
**Pull request by eldeal**
### What

Updated documentation for missed config
Change config logging to hide secrets but log all other config

### How to review

Check removing tokens hasn't broken anything
Check documentation changes make sense
Check no secrets are logged

### Who can review

Anyone

### CODE REVIEW

**Comment by eldeal** [comment]
Yes that's exactly what I meant, and I had forgotten one crucial part in making that actually secure so it should actually work now 🤦‍♀️ 

For reference, basically, on startup we log the whole Config struct exactly as you've said BUT the struct tags `json:"-"` that exist on certain fields should mean they're omitted from that output

The JSON marshaller ignores those fields, and _because there's a String() method_ (which is the bit i forgot) on the Config struct when you pass it to log.Data and it gets output as a string it should end up as the full config without those fields.

TL;DR - run the app, check the 'this is the config' log on startup, if secrets=needs fix
